RELEASES.md

```diff
@@ -0,0 +1,37 @@
+# Version 1.2
+
+### UI Improvements
+
+A lot of UI improvements
```

Comment:
Please be more detailed, there are many changes. Improved progress bar, monaco editor, yara code completion, better status page, etc...

This snippet can be helpful to pick the ones relevant.
```
git log --pretty=oneline v1.1..HEAD
```